### PR TITLE
Advance deprecation of benchmarking functionality in std.datetime

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
Undocumented all public examples and extended the deprecation message stating that all deprecated symbols will be removed after 2.094.

### CODE REVIEW

File: std/datetime/package.d [hunk 1]
```diff
@@ -182,12 +182,13 @@ import std.typecons : Flag, Yes, No;
 
     Otherwise, if set to `AutoStart.yes`, then the stopwatch is started when
     it is constructed.
-  +/
-deprecated("Use std.datetime.stopwatch.AutoStart.") alias AutoStart = Flag!"autoStart";
+  */
+deprecated("To be removed after 2.940. Use std.datetime.stopwatch.AutoStart instead.")
```
Comment (PetarKirov) [inline std/datetime/package.d]:
```suggestion
deprecated("To be removed after 2.094. Use std.datetime.stopwatch.AutoStart instead.")
```

Comment (MoonlightSentinel) [reply std/datetime/package.d]: Done

**Comment by MoonlightSentinel** [comment]
Style passes now that `std/datetime/package.d` is excluded from `undocumented_declaration_check` (second commit)

The `vibe-d/vibe-core+epoll` failure seems unrelated

**Comment by MoonlightSentinel** [comment]
Ping

**Comment by MoonlightSentinel** [comment]
Ping

**Comment by MoonlightSentinel** [comment]
@wilzbach Removed some sneaky `/// ditto` and `///` leading deprecated unittests

**Pull request merged**